config/dev.json

```diff
@@ -296,18 +158,10 @@
       "*": {
         "*": [
           "abi",
-          "devdoc",
-          "userdoc",
           "metadata",
           "evm.bytecode",
           "evm.deployedBytecode",
-          "storageLayout",
-          "evm.bytecode.sourceMap",
-          "evm.bytecode.linkReferences",
-          "evm.bytecode.generatedSources",
-          "evm.deployedBytecode.sourceMap",
-          "evm.deployedBytecode.linkReferences",
-          "evm.deployedBytecode.generatedSources"
```

Comment:
Aren't we going to need a config file that includes all this stuff for docgen?

Reply:
Ya, the prod config file.